JiscSD/octopus
# OCT-324

The purpose of this PR was to fix the dark mode issues that affected the main text editor on publications. 

### Tests:

I've mostly focused on fixing what was raised in the ticket and anything else I could find that broke, try and use the features in the editor to look for anything else that could have styling conflicts

To recreate the issue follow these steps:

1. Open draft publication
2. Add main text with table and link formatting (templates available)
3. Apply dark mode
4. Check styling of tables and links (see screenshots)

### Screenshots:

**Before**
![old-dark-mode](https://user-images.githubusercontent.com/69204118/185930195-15190447-86db-45a9-afbe-41d0d560f73d.png)

**New**
![new-dark-mode](https://user-images.githubusercontent.com/69204118/185930247-0fee9791-44e7-4a74-85b8-edc51d5ff278.png)
